### ISSUE TITLE
CS8600, CS8604 warnings not emmited

### ISSUE THREAD

**Issue by igoichuk** [OmniSharp/omnisharp-roslyn#1412]
`dotnet --info`
```
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100-preview-010184
 Commit:    c57bde4593

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.13
 OS Platform: Darwin
 RID:         osx.10.13-x64
 Base Path:   /usr/local/share/dotnet/sdk/3.0.100-preview-010184/

Host (useful for support):
  Version: 3.0.0-preview-27324-5
  Commit:  63a01b08e5

.NET Core SDKs installed:
  2.2.104 [/usr/local/share/dotnet/sdk]
  3.0.100-preview-010184 [/usr/local/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.2 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.2 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview-19075-0444 [/usr/local/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.2.2 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0-preview-27324-5 [/usr/local/share/dotnet/shared/Microsoft.NETCore.App]
```

VSCode Version 1.31.1 (1.31.1)
C# extension version 1.18.0-beta7

**Expected Behaviour:** show two warnings from the build output in the code editor (and in Problems pane).

**Actual Behaviour:** no warnings displayed

<img width="1316" alt="screen shot 2019-02-26 at 17 38 25" src="https://user-images.githubusercontent.com/2502071/53434321-42f71700-39ee-11e9-81af-e06ddcd5c0c0.png">

**Comment by filipw** [comment]
This was resolved by https://github.com/OmniSharp/omnisharp-roslyn/pull/1404
Could you please try on VS Code extension [v1.18.0-beta8])(https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.18.0-beta8) or by pulling latest omnisharp by setting "omnisharp.path": "1.32.11" or "omnisharp.path": "latest".
